### PR TITLE
Initial work for Fluent-bit sidecar

### PR DESCRIPTION
**Pull request by leahnp**
Basic deployment of Fluent-bit as a sidecar running alongside an application. Currently reading logs from a named directory and forwarding to elasticsearch.

### CODE REVIEW

**Comment by mikeln** [comment]
my only brief glance...   need to get the images out of your private quay repo and into the cnct repo.

File: fluent-bit-sidecar.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: log-app
+  labels:
+    app: log-app
+spec:
+  containers:
+  - name: log-app
+    image: quay.io/leahnp/leah_log_app:latest
```
Comment (coffeepac) [inline fluent-bit-sidecar.yaml]: general question:  should we be creating a samsung-cnct 'demo' github project where we can put the apps we use for things like this?  I've done this myself with some things but I'm not sure how we should be structuring it.

File: fluent-bit-sidecar.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    ports:
+    - containerPort: 8080
+    volumeMounts:
+    - name: log-storage
+      mountPath: /var/log
+  - name: sidecar-log-collector
+    image: leahnp/fbsidecar
```
Comment (coffeepac) [inline fluent-bit-sidecar.yaml]: I think this is the logging container, this should be hosted in quay.io/samsung-cnct

File: fluent-bit-sidecar.yaml (hunk 1 above)
Comment (leahnp) [inline fluent-bit-sidecar.yaml]: Ya, I didn't know what would be easier: using my demo app or <YOUR APP> everywhere. Let me know if you want to chat more about a demo repo or if you want me to lift out my app.

**Comment by leahnp** [comment]
@coffeepac do you want me to create an issue in this repo: https://github.com/samsung-cnct/kraken-ci-jobs or in this sidecar repo?

**Comment by leahnp** [comment]
Create issue: https://github.com/samsung-cnct/k2-logging-fluent-bit-sidecar/issues/4 and moved current image to cnct quay

File: fluent-bit-sidecar.yaml (hunk 2 above)
Comment (leahnp) [inline fluent-bit-sidecar.yaml]: https://quay.io/repository/samsung_cnct/k2-logging-fluent-bit-sidecar

File: fluent-bit-sidecar.yaml (hunk 1 above)
Comment (coffeepac) [inline fluent-bit-sidecar.yaml]: for now I'm fine with this being in not samsung-cnct.  but there should be an issue in this repo to decide where to host demo apps and then migrate them.

Comment (leahnp) [inline fluent-bit-sidecar.yaml]: https://github.com/samsung-cnct/k2-logging-fluent-bit-sidecar/issues/5

**Pull request merged**